Content.Shared/GameObjects/Components/Mobs/SharedOverlayEffectsComponent.cs

```diff
@@ -13,21 +21,70 @@ public abstract class SharedOverlayEffectsComponent : Component
         public sealed override uint? NetID => ContentNetIDs.OVERLAYEFFECTS;
     }
 
-    public enum ScreenEffects
+    [Serializable, NetSerializable]
+    public class OverlayContainer
     {
-        None,
-        CircleMask,
-        GradientCircleMask,
+        [ViewVariables(VVAccess.ReadOnly)]
+        public string ID { get; }
```

Comment:
I'm not sure it matters, but you might be able to use an enum here instead.

Reply:
I think I will add an enum to Content.Shared so the server and client can agree on overlay ID's properly. Caveat being when someone adds an overlay they have to also add an entry to the enum. I'll probably add a test or something to check for that.